src/promptflow/tests/executor/e2etests/test_traces.py

```diff
@@ -289,11 +320,14 @@ def assert_otel_traces(self, flow_file):
             assert isinstance(span.name, str)
             assert span.attributes["line_run_id"] == line_run_id
             assert span.attributes["framework"] == "promptflow"
-            expected_span_type = TraceType.FUNCTION
             if span.parent is None:
```

Comment:
Maybe we could extract a function for this?

Reply:
Done.